backend/official_plugins/tracim_backend_video_alert/REAMDE.md

```diff
@@ -0,0 +1,39 @@
+# Video Alert Plugin (tracim_backend_video_alert)
+
+This plugin sends a comment on a video file when it is uploaded to a space, and it is not compatible with the player.
+
+It works by checking the mimetype of the file and comparing it to a whitelist and a blacklist.
+If the mimetype is in the blacklist, a comment is added to the file. The whitelist is used to bypass the blacklist.
+Both lists are comma-separated lists of mimetypes. They can be used as a prefix system (i.e. `video/` will match all video files).
+
+## Installation
+```bash
+# from backend directory
+mkdir plugins
+cd plugins
+ln -s ../official_plugins/tracim_backend_video_alert .
+```
```

Comment:
Cette partie est déjà disponible dans le readme sur l'utilisation des plugins ;)

Reply:
I know but it was specifically requested. I do think that it's the responsibility of plugins to document themself, this way, there is no difference for an admin between official and non officials: "just look in the plugin folder for the documentation"

Reply:
Something wrong in github review.

The description of official plug need to be in the readme in official plugin folder. Not in each plugin folder.
So you can just add what is necessary in the readme already exist :) (its better